packages/plugin-ext/src/plugin/tree/tree-views.ts

```diff
@@ -62,8 +61,8 @@ export class TreeViewsExtImpl implements TreeViewsExt {
         return this.getTreeView(treeViewId).handleDrop!(treeItemId, dataTransferItems, token);
     }
 
-    protected toTreeItem(treeViewItemRef: TreeViewItemReference): any {
-        return this.treeViews.get(treeViewItemRef.viewId)?.getTreeItem(treeViewItemRef.itemId);
+    protected toTreeElement(treeViewItemRef: TreeViewItemReference): any {
```

Comment:
Is there a reason for this name change? In principle, it's breaking, so if it isn't necessary, may be best to stick with the `item` name.

Reply:
Yes, the method is simply misnamed. It does not convert to a TreeItem, it converts to the underlying domain element. I don't really see this class as part of the Theia API: first, it's part of the VS Code API implementation, which is determined by external factors, second, it's not something you can replace with inversify. So for me, the correct name trumps any compatibility concerns.

Reply:
In that case, please add a notice to the changelog indicating the potential breakage.